sites/partners/src/listings/PaperListingForm/sections/ApplicationDates.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react"
+import { useFormContext } from "react-hook-form"
+import { t, GridSection, DateField, TimeField } from "@bloom-housing/ui-components"
+import { FormListing } from "../index"
+import moment from "moment"
+
+type ApplicationDatesProps = {
+  listing?: FormListing
+}
+
+const ApplicationDates = ({ listing }: ApplicationDatesProps) => {
+  const formMethods = useFormContext()
+
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { register, watch } = formMethods
+
+  return (
+    <div>
+      <GridSection grid={false} separator>
+        <span className="form-section__title">{t("listings.sections.applicationDatesTitle")}</span>
+        <span className="form-section__description">
+          {t("listings.sections.applicationDatesSubtitle")}
+        </span>
```

Comment:
I think it would be better to use the `title` prop and create a `description` prop (currently we have `subtitle`, but it has different text styles).

Reply:
Added a new description prop and made sure it has the right styles & updated all the sections 👍 thanks!